ogc/src/main/scala/geotrellis/server/ogc/source/GeoTrellisOgcSource.scala

```diff
@@ -0,0 +1,64 @@
+package geotrellis.server.ogc.source
+
+import java.time.ZonedDateTime
+
+import cats.data.{NonEmptyList => NEL}
+import geotrellis.server.ogc.OgcTimeInterval
+import geotrellis.server.ogc.style.OgcStyle
+import geotrellis.store.{
+  GeoTrellisPath,
+  GeoTrellisRasterSourceLegacy,
+  LayerHeader
+}
+import jp.ne.opt.chronoscala.Imports._
+
+case class GeoTrellisOgcSource(
+    name: String,
+    title: String,
+    sourceUri: String,
+    defaultStyle: Option[String],
+    styles: List[OgcStyle],
+    timeMetadataKey: String = "times"
+) extends RasterOgcSource {
+
+  private val dataPath = GeoTrellisPath.parse(sourceUri)
+
+  // If layer is temporally enabled, default time is the first in the layer
+  lazy val source =
+    GeoTrellisRasterSourceLegacy(dataPath, timeInterval.map(_.start))
+
+  lazy val times: Option[NEL[ZonedDateTime]] = {
```

Comment:
Optional NonEmptyList, so is that like a list?
Didn't trace the semantics of this this but the signature made me laugh.

Reply:
`List` 💯

Reply:
Switched to `List`